<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="KComboBox" name="SortBy">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>filter by</string>
    </property>
   </item>
  </widget>
  <widget class="KComboBox" name="LabelBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>labels</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="SearchTextBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="KRichTextEdit" name="DisplayResultWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="ListResultWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBoxLogs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>logs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBoxContact">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>contacts</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBoxDate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>date</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBoxExhaustive">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>exhaustive</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
